theforeman.org/pipelines/vars/foreman/3.9.groovy

```diff
@@ -1,12 +1,11 @@
 def foreman_version = '3.9'
 def git_branch = "${foreman_version}-stable"
-def stage_source = 'copr'
+def stage_source = 'stagingyum'
 
 def foreman_client_distros = [
     'el9',
     'el8',
-    'el7',
-    'sles12'
```

Comment:
I didn't realize we'd effectively drop sles12 this way. I wonder if we could somehow restore this at some point, like syncing from OBS.

Reply:
copr can build for sles, so we could… just build it, properly.

Reply:
It's not listed in nightly, which is the source for this:
https://github.com/theforeman/jenkins-jobs/blob/b993393b01c86082cefda9cfea5dfbba5abd0d3c/theforeman.org/pipelines/vars/foreman/nightly.groovy#L5-L9